### PR TITLE
Show why list permission should never be given if 'GET' isn't

### PR DESCRIPTION
**Pull request by RichardoC**
Currently it's possible to give the LIST permission, without granting GET. Due to this some Kubernetes administrators may be under the impression that those items cannot be obtained by users or service accounts with only the LIST permission. 

This is incorrect, and I'd like to add this example to show why this shouldn't be done.

I'm happy to move this elsewhere in the documentation!

### CODE REVIEW

File: content/en/docs/reference/access-authn-authz/rbac.md [hunk 1]
```diff
@@ -929,6 +929,66 @@ To bootstrap initial roles and role bindings:
 * Use a credential with the "system:masters" group, which is bound to the "cluster-admin" super-user role by the default bindings.
 * If your API server runs with the insecure port enabled (`--insecure-port`), you can also make API calls via that port, which does not enforce authentication or authorization.
 
+## Preventing excess access via the list permission
```
[the remark this answers is not in the document]
Comment (sftim) [reply content/en/docs/reference/access-authn-authz/rbac.md:932]: It'd be better to have the reference docs link elsewhere (eg to the RBAC good practice guide that's WIP). Reference docs _should_ be terse and somewhat formal.

[the remark this answers is not in the document]
Comment (RichardoC) [reply content/en/docs/reference/access-authn-authz/rbac.md:932]: Where's the RBAC good practice doc? Happy to contribute this there and link over to that doc?

[the remark this answers is not in the document]
Comment (sftim) [reply content/en/docs/reference/access-authn-authz/rbac.md:932]: https://kubernetes.io/docs/concepts/security/rbac-good-practices/

**Comment by zacharysarah** [comment]
@RichardoC 👋🏻 Please feel free to `/reopen` when you're ready to continue.

/close

**Pull request closed without merging**